core/tests/webdriverio_utils/ExplorationPlayerPage.js

```diff
@@ -105,8 +106,13 @@ var ExplorationPlayerPage = function() {
   };
 
   this.clickThroughToNextCard = async function() {
-    await waitFor.elementToBeClickable(nextCardButton);
-    await action.click('Next Card button', nextCardButton);
+    await waitFor.elementToBeClickable(continueToNextCardButton);
+    await action.click('Next Card button', continueToNextCardButton);
+  };
+
+  this.clickNextButton = async function() {
+    await waitFor.elementToBeClickable(nextButton);
+    await action.click('Next Card button', nextButton);
```

Comment:
action.click will internally ensure `elementToBeClickable`, right?

Reply:
Yeah, didn't check it. Done